developers/guides/access-control-jwt.mdx

```diff
@@ -317,3 +317,29 @@ URL as a query parameter, similar to:
 ```
 https://playback.livepeer.studio/asset/hls/abcd1234/index.m3u8?jwt=your-jwt
 ```
+
+If you prefer not to use query parameters to pass the JWT, 
+it can alternatively be transmitted as a header. To accomplish this, 
+you must set a header named `Livepeer-JWT` with your access key as its value.
+
+Below is an example of how to use headers when using 
+the Livepeer Player:
+
+```tsx
+import { Player } from "@livepeer/react";
+
+const hlsConfig = {
+  xhrSetup: function(xhr, url) {
+    xhr.setRequestHeader('Livepeer-Jwt', 'YOUR_JWT');
```

Comment:
I'll update this but this shouldn't be necessary now. Let's not merge until this change is made.